4.processing_image_based_profiles/scripts/merge_feature_parquets.py

```diff
@@ -108,6 +108,65 @@
 # In[4]:
 
 
+# create a record for each compartment
+merged_df_dict = {
+    "Organoid": {
+        "AreaSize_Shape": [],
+        "Colocalization": [],
+        "Intensity": [],
+        "Granularity": [],
+        "Neighbor": [],
+        "Texture": [],
+    },
+    "Cell": {
+        "AreaSize_Shape": [],
+        "Colocalization": [],
+        "Intensity": [],
+        "Granularity": [],
+        "Neighbor": [],
+        "Texture": [],
+    },
+    "Nuclei": {
+        "AreaSize_Shape": [],
+        "Colocalization": [],
+        "Intensity": [],
+        "Granularity": [],
+        "Neighbor": [],
+        "Texture": [],
+    },
+    "Cytoplasm": {
+        "AreaSize_Shape": [],
+        "Colocalization": [],
+        "Intensity": [],
+        "Granularity": [],
+        "Neighbor": [],
+        "Texture": [],
+    },
+}
+for compartment in feature_types_dict.keys():
```

Comment:
There might be an opportunity to simplify using SQL-based path globbing within DuckDB using the [SQLite extension](https://duckdb.org/docs/stable/core_extensions/sqlite.html). A nice part here would be that if/when you decide to lean into DuckDB further (instead of SQLite) that the design would be nearly already finished.